### DISCUSSION TITLE
AttributeError: module 'rdkit.Chem' has no attribute 'GraphDescriptors'

### DISCUSSION THREAD

**Discussion by mdingm** [rdkit/rdkit#6616] [FAQ]
Hi rdkit! First of all, I would like to express my gratitude to you for doing such excellent work in your development, which has provided me with a lot of help in my research.

However, I have encountered some problems: In the rdkit 2023.03.1 version of the [official document](https://www.rdkit.org/docs/source/rdkit.Chem.GraphDescriptors.html), Chem.GraphDescriptors module exists. But when I used the following code and it didn't work:
`smiles= "C1=CC(C(CC)(C/C=C/C2=CC=CC=C2)N(C)CC2CC2)=CC=C1" ` 
`mol = Chem.MolFromSmiles(smiles)`
`rdkit.Chem.GraphDescriptors(mol)`

Run the above code, I get the **AttributeError: module 'rdkit.Chem' has no attribute 'GraphDescriptors'.** 

By running `rdkit.__version__` to see the lastest rdkit, I got the version: 2023.03.2

And I tried many times to uninstall rdkit and install it, it doesn't work. If you can help me solve the above problems, I will be very grateful! Hope to get your attention soon. 

Best wishes for you.

**Comment by greglandrum** [comment]
@mdingm GraphDesciptors is a python module, not a function.
Try doing something like this:
```
smiles= "C1=CC(C(CC)(C/C=C/C2=CC=CC=C2)N(C)CC2CC2)=CC=C1" 
mol = Chem.MolFromSmiles(smiles)
from rdkit.Chem import GraphDescriptors
GraphDescriptors.Chi0(mol)
```
